libs/helperHashEncode.d.ts

```diff
@@ -1,13 +1,13 @@
 export default hash;
 declare namespace hash {
-    export { encode };
+  export { encode };
 }
 /**
  * Helper function to hash encode personal number
  *
- * @param {number} number Personal number to be hashed
+ * @param {unknown} number Personal number to be hashed
```

Comment:
Might as well remove the entire function comment since it seems misleading and redundant.